pandas/core/strings/accessor.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import annotations
 
 import codecs
+from collections.abc import Callable  # noqa: PDF001
```

Comment:
we should disable the flake rule. 

we should be importing Callable from collections.abc
https://docs.python.org/3/library/typing.html#callable